setup.sql

```diff
@@ -44,4 +44,12 @@ CREATE TABLE account (
 	first_name TEXT NOT NULL,
 	last_name TEXT NOT NULL,
 	account_type TEXT NOT NULL
-)
+);
+
+CREATE TABLE business_information (
+	id SERIAL PRIMARY KEY,
+	business_name VARCHAR(50) NOT NULL,
+	owner_id INT NOT NULL,
```

Comment:
i don't think we need an owner_id, but we could add in a phone number and address column here

Reply:
Okay added!